projects/rocprofiler-sdk/source/bin/rocprofv3.py

```diff
@@ -437,6 +437,44 @@ def add_parser_bool_argument(gparser, *args, **kwargs):
         nargs="*",
     )
 
+    spm_options = parser.add_argument_group("Streaming Perfmon options")
+
+    add_parser_bool_argument(
+        spm_options,
+        "--spm-beta-enabled",
+        help="enable SPM; beta version",
+    )
+
+    spm_options.add_argument(
+        "--spm",
+        help=(
+            "Specify SPM events to collect(comma OR space separated in case of more than 1 counters). "
+            "Note: job will fail if entire set of counters cannot be collected in single pass"
+        ),
+        default=None,
+        nargs="*",
+    )
+
+    spm_options.add_argument(
+        "--spm-buffer-size",
+        help="SPM Buffer size",
+        default=None,
+        type=str,
+    )
+
+    spm_options.add_argument(
+        "--spm-timeout",
+        help="Timeout for SPM, in ms. Larger values may slow down the application, while smaller values may drop data. Default = 30.",
+        default=None,
```

Comment:
I thought the default is 30

```suggestion
        default=30,
```

Reply:
The default are in config.hpp. Do you recommend I have it in a comment in rocprofv3.py?

Reply:
I think so that would be good. Any reason why not to? Because we might change the default value in the future? If you dislike hardcoding the default interval in the comment here, I think it's worth saying at least where do we read the default values from (here or in the documentation). I'm pretty sure that the users would like to know the value of the default interval in case they want to change, so they know if they need to increase/decrease it.

Reply:
Done